src/iaux-underlined-tab-bar-loading-dots.ts

```diff
@@ -0,0 +1,75 @@
+import { html, css, LitElement, customElement, CSSResult } from 'lit-element';
+
+@customElement('iaux-underlined-tab-bar-loading-dots')
+export class LoadingDots extends LitElement {
+  render() {
+    return html`
+      <div class="spinnyCircle">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+    `;
+  }
+
+  static get styles(): CSSResult {
+    const loadingDotColorCSS = css`var(--tabBarLoadingDotColor)`;
+    return css`
+      .spinnyCircle {
+        display: block;
+        position: relative;
+        width: 58px;
```

Comment:
would it be useful to be able to customize this size?

Reply:
Thanks for the suggestion, the loading state size is now dynamic using CSS variables.